lib/appoptics_apm/frameworks/padrino.rb

```diff
@@ -75,6 +76,7 @@ def render_with_appoptics(engine, data = nil, options = {}, locals = {}, &block)
             begin
               render_without_appoptics(engine, data, options, locals, &block)
```

Comment:
This question has nothing to do with your changes, but is `render_without_appoptics()` padrino's original method and `render_with_appoptics()` is the method that replaces it using some kind of ruby magic like `prepend`?

Reply:
I does a so-called alias chain, used by almost all of our instrumentation. 
see lib/appoptics_apm/util.rb:28 , mainly:
```
cls.class_eval do
              alias_method without_appoptics, method.to_s
              alias_method method.to_s, with_appoptics
            end
```
`prepend` is a newer way to do it, but not backward compatible enough (yet).